lib/iris/tests/results/netcdf/netcdf_save_gridmapmulti.cdl

```diff
@@ -15,12 +15,12 @@ variables:
 		latitude_longitude:earth_radius = 6371229. ;
 	int64 longitude(longitude) ;
 		longitude:axis = "X" ;
-		longitude:units = "degrees_east" ;
+		longitude:units = "1" ;
```

Comment:
i'm surprised by the test result changes in this file.  I would have expected all of these to stay the same

@djkirkham why do you think this has occurred?